docs/book/driver-deployment/deploying_csi_with_zones.md

```diff
@@ -132,13 +132,12 @@ Install the vSphere CPI and the CSI driver using the zone and region entries.
    ```bash
    cd /etc/kubernetes
    kubectl create configmap cloud-config --from-file=vsphere.conf --namespace=kube-system
-
-   kubectl apply -f https://raw.githubusercontent.com/kubernetes/cloud-provider-vsphere/master/manifests/controller-manager/cloud-controller-manager-roles.yaml
-   kubectl apply -f https://raw.githubusercontent.com/kubernetes/cloud-provider-vsphere/master/manifests/controller-manager/cloud-controller-manager-role-bindings.yaml
-   kubectl apply -f https://github.com/kubernetes/cloud-provider-vsphere/raw/master/manifests/controller-manager/vsphere-cloud-controller-manager-ds.yaml
    ```
+   
+2. Deploy the `vsphere-cloud-controller-manager.yaml` from the latest version of [vSphere CPI releases](https://github.com/kubernetes/cloud-provider-vsphere/tree/master/releases)
```

Comment:
Shouldnt this also be qualified? What i mean is, we should suggest installing a CPI version that FVT/ST qualifies in their testing.

Reply:
You are right, I have removed that change. We will qualify it first and then change the documentation.